ci-operator/step-registry/openshift-qe/workers-infra-workload/openshift-qe-workers-infra-workload-commands.sh

```diff
@@ -0,0 +1,698 @@
+kind: MachineConfigPool
+metadata:
+  name: infra
+  labels:
+    operator.machineconfiguration.openshift.io/required-for-upgrade: ''
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - {key: machineconfiguration.openshift.io/role, operator: In, values: [worker,infra]}
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/infra: ""
+EOF
+ elif [[ $MACHINESET_TYPE == "workload" ]];then
+      oc apply -f- <<EOF
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: workload
+  labels:
+    operator.machineconfiguration.openshift.io/required-for-upgrade: ''
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - {key: machineconfiguration.openshift.io/role, operator: In, values: [worker,workload]}
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/workload: ""
+EOF
+ else
+     echo "Invalid machineset type, should be [infra] or [workload]"
+ fi
+}
+
+
+######################################################################################
+#                                                                                    #
+#                       Add Infra and Workload Entrypoint                            #
+#                                                                                    #
+######################################################################################
+#SET_ENV_BY_PLATFORM can be overried by SET_ENV_BY_PLATFORM=custom, it will use the same cpu/ram with worker nodes
+#IF_INSTALL_INFRA_WORKLOAD=true/false
+if test ! -f "${KUBECONFIG}"
+then
+	echo "No kubeconfig, can not continue."
+	exit 0
+fi
+
+IF_INSTALL_INFRA_WORKLOAD=${IF_INSTALL_INFRA_WORKLOAD:=true}
+if [[ ${IF_INSTALL_INFRA_WORKLOAD} != "true" ]];then
+   echo "No need to install infra and workload for this OCP cluster"
+   exit 1
+fi
+
+# Download jq
+if [ ! -d /tmp/bin ];then
+  mkdir /tmp/bin
+  export PATH=$PATH:/tmp/bin
+  curl -sL https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 > /tmp/bin/jq
+  chmod ug+x /tmp/bin/jq
+fi
+
+#Get Basic Infrastructue Architecture Info
+node_arch=$(oc get nodes -ojsonpath='{.items[*].status.nodeInfo.architecture}')
+platform_type=$(oc get infrastructure cluster -ojsonpath='{.status.platformStatus.type}')
+platform_type=$(echo $platform_type | tr -s 'A-Z' 'a-z')
+node_arch=$(echo $node_arch | tr -s " " "\n"| sort -u)
+all_machinesets=$(oc -n openshift-machine-api get machineset -ojsonpath='{.items[*].metadata.name}{"\n"}')
+machineset_list=$(echo $all_machinesets | tr -s ' ' '\n'| sort -u| grep -v -i -E "infra|workload|win"| head -n3)
+machineset_count=$(echo $all_machinesets | tr -s ' ' '\n'| sort -u| grep -v -i -E "infra|workload|win"| head -n3 |wc -l)
+total_worker_nodes=$(oc get nodes -l node-role.kubernetes.io/worker= -oname|wc -l)
+
+scale_type=""
+#Currently only support AWS reference to ROSA settings
+if [[ $total_worker_nodes -gt 100 ]];then
+	scale_type=large
```

Comment:
These should show as medium (GT 100), small (between 26 and 100), and then extra small (1 to 25) to keep aligned with the sizing/names of the perfscale dev and qe teams 

The if statements below will have to change as well

Reply:
Fixed in new code, please review again, thanks!